### PR TITLE
SCALc5ea8337-SCALefbe78e7-SCAU0fc6711f-SCAU23215b11- SCAU29eb6c15

### PR DESCRIPTION
**Pull request by OdeclasV**
more new CA yamls

### CODE REVIEW

File: members/SCALc5ea8337.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          options:
+            max_length: 2000
+          required: true
+    - click_on:
+        - value: Submit
+          selector: "#submitButton"
+    - wait:
+        - value: 1
+    - find:
+        - selector: "tr:nth-child(3)"
```
Comment (tomdionesotes) [inline members/SCALc5ea8337.yaml]: This particular selector in this `find` was useful for another CA legislator's form, does it correctly find the element with the thank you text for this form as well?

File: members/SCAU0fc6711f.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          selector: "#submitButton"
+    - wait:
+        - value: 1
+    - find:
+        - selector: "tr:nth-child(3)"
+  success:
+    headers:
+      status: 200
+    body:
+      excludes: "Select an Issue"
```
Comment (tomdionesotes) [inline members/SCAU0fc6711f.yaml]: Is the thank you message for this form just "null" as well?

Comment (OdeclasV) [reply members/SCAU0fc6711f.yaml]: si, [Screenshot from Gyazo](https://gyazo.com/ac3b0068caf5dec7be62fc138522ff0c)

File: members/SCALc5ea8337.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SCALc5ea8337.yaml]: Yeah, this is actually the one we were working on last week

https://s3.amazonaws.com/yamltron/screenshots/final/3c69a545-0253-404d-93a6-32d202511ac5.png

**Pull request merged**